core/src/test/java/org/frankframework/senders/MessageStoreSenderInputProcessingTest.java

```diff
@@ -12,7 +12,7 @@
 import org.frankframework.stream.Message;
 import org.junit.jupiter.api.Test;
 
-public class MessageStoreSenderTest extends SenderTestBase<MessageStoreSender> {
+public class MessageStoreSenderInputProcessingTest extends SenderTestBase<MessageStoreSender> {
```

Comment:
I renamed this class to avoid confusion with the real MessageStoreSenderTest that I added.